afterpay/src/test/kotlin/com/afterpay/android/AfterpayEnabled.kt

```diff
@@ -1,12 +1,22 @@
 package com.afterpay.android
 
+import com.afterpay.android.internal.Locales
 import org.junit.Assert
 import org.junit.Test
 import java.util.Locale
 
 class AfterpayEnabled {
     private val environment = AfterpayEnvironment.SANDBOX
 
+    private val validMerchantLocales: Array<Locale> = arrayOf(
+        Locales.EN_AU,
+        Locales.EN_CA,
+        Locales.EN_GB,
+        Locales.EN_US,
+        Locales.EN_NZ,
```

Comment:
Is this meant to be alphabetical order?
```suggestion
        Locales.EN_NZ,
        Locales.EN_US,
```

Reply:
Yes it should have been. Good pickup.